project/ScalazBuild.scala

```diff
@@ -45,7 +46,8 @@ object Scalaz {
       compilerPlugin("com.github.tomasmikula" %% "pascal"          % "0.2.1"),
       compilerPlugin("com.github.ghik"        %% "silencer-plugin" % "0.6")
     ),
-    incOptions ~= (_.withLogRecompileOnMacro(false))
+    incOptions ~= (_.withLogRecompileOnMacro(false)),
+    maxErrors := 5
```

Comment:
Why this one?

Reply:
oops, forgot to take it back out, the output was overwhelming initially :D